app/src/main/res/layout/fragment_onboarding_welcome.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
     <com.airbnb.lottie.LottieAnimationView
         android:id="@+id/animation_view"
-        android:layout_width="256dp"
-        android:layout_height="256dp"
-        android:layout_marginBottom="28dp"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
```

Comment:
This makes the animation smaller, even when there is enough room available (i.e. with normal font size). Could you rather try to make it adaptive (something like "up to 256 dp when enough room is available, otherwise fitting the parent with ~64dp margin to the sides"?)

Reply:
Thank you so much for your advice. Yes, using wrap_content will make the animation keep its own size, so I suppose it would be better to simply set the width of animation from `android:layout_width="wrap_content"` to `android:layout_width="fill_parent"`. In the meantime, I would like to set the width of ConstraintLayout which surrounding this animation to fill_parent, which will make the width of  ConstraintLayout much wider. According to this, I have made some new changes. I would like to commit them later.

So this is the screenshot in small font size after my latest changes:

![Screenshot_1628844636](https://user-images.githubusercontent.com/25502419/129331397-4645c84a-f504-4427-ae3e-5cd7f26c4c39.png)

and the screenshot in largest font size:

![Screenshot_1628844651](https://user-images.githubusercontent.com/25502419/129331468-ec50af73-e370-4091-af0a-50d96134c8d3.png)

Reply:
I just realized that the size of this animation is not changed after all. But I guess the size is good enough for display, because considering consistency, if I magnify this animation so the animation after this page should be magnified as well, which is too wide to be any bigger. This is my thought.